src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <a href="#home"
+          class="logo_home">
+          <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+          >
+        </a>
```

Comment:
is this link an element of header? Use BEM
```suggestion
        <a href="#home"
          class="header__logo-link">
          <img
          class="header__logo-img"
          src="images/logo.png"
          alt="Moyo"
          >
        </a>
```